samcli/hook_packages/terraform/hooks/prepare/property_builder.py

```diff
@@ -204,6 +204,7 @@ def _check_image_config_value(image_config: Any) -> bool:
     "Runtime": _get_property_extractor("runtime"),
     "Layers": _get_property_extractor("layers"),
     "Timeout": _get_property_extractor("timeout"),
+    "MemorySize": _get_property_extractor("memory_size"),
```

Comment:
Will there be `memory_size` property defined all the time, or should we add this conditionally? 
cc: @moelasmar

Reply:
it is Optional for the customer, but has a default value. The generated plan will always contains a value.